app/index.js

```diff
@@ -32,9 +32,11 @@ passport.use(new Strategy({
     domain: process.env.AUTH0_DOMAIN,
```

Comment:
Mind you... if we're changing `AUTH0` to `OIDC` in the CP app, shouldn't we do the same here?

Reply:
yes

Reply:
👍